content/en/docs/installation/cli.md

```diff
@@ -4,27 +4,89 @@ description: Install the notation CLI on Linux, macOS, and Windows
 weight: 1
 ---
 
-## Download the `notation` CLI 
+## Download the `notation` CLI
 
-Download the latest CLI for your platform from the [Releases page](https://github.com/notaryproject/notation/releases).
+Navigate to the [Releases page](https://github.com/notaryproject/notation/releases) and select the latest release of the CLI. The notation CLI binaries for each platform are available under the *Assets* section in the `notation_<version>_<platform>.tar.gz` files.
```

Comment:
From a user's perspective, it's inconvenient for me to just copy and paste to finish the installation step. I would suggest providing download & installation commands and consolidating them into different tabs.  See the [Minikube getting start page](https://minikube.sigs.k8s.io/docs/start/) as an example:

![image](https://user-images.githubusercontent.com/40452856/203216827-dddb7130-e148-4a76-9484-c435a95f1ae4.png)

Reply:
This format can be improved later after I confirmed with @toddysm . I will create an issue to track it.